### PR TITLE
Stick to single quote convention + minor changes

### PR DESCRIPTION
**Pull request by eexit**
Thank you very much for the fix on #236, really and thank you more the reactivity.

This small PR brings an alternative to single quote "convention" break and bring back the single quote. I also made minor language-wise changes.

### CODE REVIEW

File: PhpAmqpLib/Helper/MiscHelper.php [hunk 1]
```diff
@@ -67,23 +67,23 @@ public static function hexdump($data, $htmloutput = true, $uppercase = false, $r
...
-        $x = ($uppercase === false) ? 'x' : 'X';
+        $hexFormat = $uppercase ? 'X' : 'x';
 
         // Iterate string
         for ($i = $j = 0; $i < $len; $i++) {
             // Convert to hexidecimal
-            $hexi .= sprintf("%02$x ", ord($data[$i]));
+            // We must use concatenation here because the $hexFormat value
+            // is needed for sprintf() to parse the format
+            $hexi .= sprintf('%02' .  $hexFormat . ' ', ord($data[$i]));
```
Comment (videlalvaro) [inline PhpAmqpLib/Helper/MiscHelper.php]: why not sprintf the `$hexFormat` variable as well?

File: PhpAmqpLib/Helper/MiscHelper.php [hunk 2]
```diff
@@ -97,7 +97,9 @@ public static function hexdump($data, $htmloutput = true, $uppercase = false, $r
             // Add row
             if (++$j === 16 || $i === $len - 1) {
                 // Join the hexi / ascii output
-                $dump .= sprintf("%04$x  %-49s  %s", $offset, $hexi, $ascii);
+                // We must use concatenation here because the $hexFormat value
+                // is needed for sprintf() to parse the format
+                $dump .= sprintf('%04' . $hexFormat . '  %-49s  %s', $offset, $hexi, $ascii);
```
Comment (videlalvaro) [inline PhpAmqpLib/Helper/MiscHelper.php]: same as above

File: PhpAmqpLib/Helper/MiscHelper.php (hunk 1 above)
Comment (fprochazka) [inline PhpAmqpLib/Helper/MiscHelper.php]: because the variable contains the modifier type that is used for sprintf formating

Comment (videlalvaro) [inline PhpAmqpLib/Helper/MiscHelper.php]: Oh… stupid me

**Pull request merged**